ui/src/layout/Assets/AssetsTable.js

```diff
@@ -43,6 +43,18 @@ const AssetsTable = () => {
             sortIds: LOCATION_SORT_IDS,
             accessor: "assetInfo.location"
         },
+        {
+            Header: "LastSeen",
+            id: "lastSeen",
+            sortIds: ["lastSeen"],
+            accessor: original => formatDate(original.lastSeen)
+        },
+        {
+            Header: "Terminated",
```

Comment:
On a fresh run it's not clear what are the values types. I think that better to state "Terminated On"